docs/modules/ROOT/pages/tutorial/deploy.adoc

```diff
@@ -174,6 +204,59 @@ This is a contract that replicates a box, with three functions:
 * `removeObject()`: Decreases the number of objects in the box by removing one.
 * `_authorizeUpgrade()`: Points the proxy to a new implementation address.
 
+[[foundry-deploy]]
+=== Foundry
+
+. Create a file named `Deploy.s.sol` inside the `script` directory. This script will deploy the upgradeable Box contract through Defender with an initial amount of 5 objects inside and the owner as the multisig address configured in the environment setup. The `initializer` option is used to call the `initialize()` function after the contract is deployed. Copy and paste the code below into `Deploy.s.sol`:
++
+[source,jsx]
+----
+// SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.20;
+
+import {Script} from "forge-std/Script.sol";
+import {console} from "forge-std/console.sol";
+
+import {Defender, ApprovalProcessResponse} from "openzeppelin-foundry-upgrades/Defender.sol";
+import {Upgrades, Options} from "openzeppelin-foundry-upgrades/Upgrades.sol";
+
+import {Box} from "src/Box.sol";
+
+contract DefenderScript is Script {
+    function setUp() public {}
+
+    function run() public {
+        ApprovalProcessResponse memory upgradeApprovalProcess = Defender.getUpgradeApprovalProcess();
+
+        if (upgradeApprovalProcess.via == address(0)) {
+            revert(
+                string.concat(
+                    "Upgrade approval process with id ",
+                    upgradeApprovalProcess.approvalProcessId,
+                    " has no assigned address"
+                )
+            );
+        }
+
+        Options memory opts;
+        opts.defender.useDefenderDeploy = true;
+
+        address proxy =
+            Upgrades.deployUUPSProxy("Box.sol", abi.encodeCall(Box.initialize, (5, upgradeApprovalProcess.via)), opts);
+
+        console.log("Deployed proxy to address", proxy);
+    }
+}
+----
+. Deploy by running the following command which executes your deployment script:
++
+```
+forge script script/Deploy.s.sol --ffi --rpc-url https://ethereum-sepolia.publicnode.com -vvvv
```

Comment:
We can remove `--ffi` since it is configured in the "Foundry setup" step, but should recommend `--force` so that it cleans out old build files.  (I realize this is from the plugin docs, so I will make the same updates there).
```suggestion
forge script script/Deploy.s.sol --force --rpc-url https://ethereum-sepolia.publicnode.com -vvvv
```